.pug-lintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "disallowAttributeConcatenation": true,
```

Comment:
I think `disallowClassAttributeWithStaticValue`, `disallowIdAttributeWithStaticValue` and `requireLowerCaseTags` may be good to standardise too: https://github.com/pugjs/pug-lint/blob/master/docs/rules.md#disallowclassattributewithstaticvalue-true